files/ko/learn/javascript/building_blocks/build_your_own_function/index.html

```diff
@@ -1,57 +1,57 @@
 ---
-title: Build your own function
+title: 자신만의 함수 만들기
 slug: Learn/JavaScript/Building_blocks/Build_your_own_function
 translation_of: Learn/JavaScript/Building_blocks/Build_your_own_function
 ---
 <div>{{LearnSidebar}}</div>
 
 <div>{{PreviousMenuNext("Learn/JavaScript/Building_blocks/Functions","Learn/JavaScript/Building_blocks/Return_values", "Learn/JavaScript/Building_blocks")}}</div>
 
-<p class="summary">이번 장에선 앞의 글에서 배운 function이론을 사용하여 사용자 정의 함수를 만들어 봅니다. 사용자 정의 함수 연습과 더불어, 함수를 다루는 몇 가지 유용한 사항들도 설명을 하겠습니다. </p>
+<p class="summary">지난 문서에서 다뤄진 필수적인 이론의 대부분을 가지고서, 이 문서는 실제적인 경험을 제공합니다. 여기서 당신만의 사용자 정의(custom) 함수를 만드는 몇몇 연습을 얻을 것입니다. 그 과정에서, 우리는 또한 함수를 다루는 몇몇 유용한 세부 사항을 설명할 것입니다.</p>
```

Comment:
```suggestion
<p class="summary">지난 문서에서 다뤄진 필수적인 이론의 대부분을 바탕으로, 이 문서는 실제적인 경험을 제공합니다. 여기서 당신만의 사용자 정의(custom) 함수를 만드는 연습을 할 수 있습니다. 그 과정에서 우리는 함수를 다루는 몇몇 유용한 세부 사항을 설명할 것입니다.</p>
```

문장의 흐름을 이런 식으로 가져가보면 어떨까요? 😀